packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/02-chat-with-video.mdx

```diff
@@ -3,6 +3,483 @@ title: Chat Integration
 description: How to integrate chat & video
 ---
 
+It's common for calling apps to have chat, as well as the opposite - chat apps to have a calling functionality. Stream's Chat and Video SDKs are perfectly compatible between each other, and can easily be integrated into an app.
+
+Before starting with this guide on how to integrate chat and video together, we will initially focus on installation of the Chat SDK.
+
+## Installation
+
+To install Stream Chat SDK, you can go through our [Getting Started](https://getstream.io/chat/docs/sdk/reactnative/) guide and follow the installation steps and install all the relevant dependencies as well as optional dependencies according to your application's needs.
+
+:::note
+Do not forget through the [Additional Steps](https://getstream.io/chat/docs/sdk/reactnative/#additional-steps) guide while installing the dependencies.
+:::
+
+## Creating and Passing Chat Client
+
+To create the Chat Client we will mainly follow [this tutorial](https://getstream.io/chat/react-native-chat/tutorial/#add-stream-chat-to-the-application).
+
+### Wrapping the chat client
+
+To wrap our Chat initiation and client logic we will create a `ChatWrapper` component that can be wrapped at the root of the application, so that the client will be available to all the child components.
+
+```tsx title="ChatWrapper.tsx"
+import React, { PropsWithChildren, useCallback, useMemo } from 'react';
+import { Chat, OverlayProvider, Streami18n } from 'stream-chat-react-native';
+import { useChatClient } from '../hooks/useChatClient';
```

Comment:
I think we can better clarify the path names here, actually in whole tutorial

Like in audio room tutorial I think its better to include simple path names in the titles and use those in the files

Example

```
tsx title="src/hooks/useChatClient.tsx"
```

and also use it inside the code samples like

```
import { useChatClient } from 'src/hooks/useChatClient';
```

Reply:
this is important because there will be a lot of people who will just copy paste the code and expect it to work

Reply:
Done